extensions/standard-processors/processors/GetFile.cpp

```diff
@@ -52,50 +52,44 @@ namespace minifi {
 namespace processors {
 
 core::Property GetFile::BatchSize(
-    core::PropertyBuilder::createProperty("Batch Size")->withDescription("The maximum number of files to pull in each iteration")->withDefaultValue<uint32_t>(10)
-        ->build());
```

Comment:
artifact of fixing linter errors with eclipse. this is easily avoided by ignoring white space in the review.